EpikCord/__main__.py

```diff
@@ -1,4 +1,60 @@
-from EpikCord import __version__
+import argparse
+import colorama
+
+
+__all__ = ("__version__", "info")
+__version__ = "0.5.2"
+_bot_tmplate = """#!/usr/bin/env python
+from EpikCord import Client, Intents
+
+intents = Intents.all()
+
+client: Client = Client([token], intents)
+
+
+@client.event()
+async def ready():
+    print("Ready!")
+# Write commands here
+
+client.login()
+"""
+win_restricted = (
+    "CON",
+    "CON1",
+    "CON2",
+    "CON3",
+    "CON4",
+    "CON5",
+    "CON6",
+    "CON7",
+    "CON8",
+    "CON9",
+    "AUX",
+    "AUX1",
+    "AUX2",
+    "AUX3",
+    "AUX4",
+    "CON5",
+    "CON6",
+    "CON7",
+    "CON8",
+    "CON9",
+
+)
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--version", help="displays the version of the library", action="store_true"
+)
+colorama.init(autoreset=True)
```

Comment:
I dont like the fact that the cl tool install others dependencies such a colorama, because if i use something like rich, i will have 2 different deps for text coloring....

Reply:
it is a built in library for all i know

Reply:
> it is a built in library for all i know

![image](https://user-images.githubusercontent.com/53050011/180612909-86c5e2c2-2098-40d3-845a-9c152b88524a.png)

nope :)

Reply:
Yeah, not a built in and it's rather useless